jenkins/job-configs/kubernetes-jenkins-pull/charts-pull.yaml

```diff
@@ -0,0 +1,100 @@
+- job:
+    name: 'charts-pull-test-e2e'
+    concurrent: true
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+        - github:
+            url: 'https://github.com/kubernetes/charts'
+        - throttle:
+            max-total: 4
+            max-per-node: 2
+            option: project
+        - raw:
```

Comment:
I will need to download some creds for a k8s cluster so will need gcloud to be there. I have been asked to try to use docker vs installing any deps which I will try to adhere to.